arch-update/arch-update.py

```diff
@@ -27,8 +27,9 @@ def create_argparse():
     parser.add_argument(
         '-a',
         '--aur',
-        action='store_true',
-        help='Include AUR packages. Attn: Yaourt must be installed'
+        default='yaourt',
```

Comment:
This means that by default AUR updates are checked using yaourt, even if `-a` is not specified.